### CI status — blocked by a pre-existing `ci.yml` bug, verified locally instead

GitHub CI for this PR fails at the **`Override team submodule with current branch`**
step:

```
##[error]An error occurred trying to start process '/usr/bin/bash' with working
directory '/home/runner/work/akgentic-team/akgentic-team/packages/akgentic-team'.
No such file or directory
```

This is a **pre-existing infrastructure bug in `.github/workflows/ci.yml`**, unrelated
to this change — the `working-directory: packages/akgentic-team` step assumes a path
that the `submodules: recursive` checkout did not populate. It fails before mypy, ruff,
or pytest ever run. To be fixed separately (not in this PR).

#### Local verification — exact CI `quality`-job commands

| Gate | Command | Result |
|------|---------|--------|
| Type check | `mypy packages/akgentic-team/src/` | ✅ Success — no issues, 20 files |
| Lint | `ruff check packages/akgentic-team/src/` | ✅ All checks passed |
| Tests + coverage | `pytest packages/akgentic-team/tests/ --cov=akgentic.team --cov-fail-under=80` | ✅ 350 passed, 5 skipped, 1 deselected — **93.94%** coverage |

The 5 skips / 1 deselect are pre-existing and unrelated to this change.

Once the `ci.yml` fix lands, this branch should be rebased so CI runs clean.

--- a/src/akgentic/team/messages.py
+++ b/src/akgentic/team/messages.py
@@ -22,8 +22,9 @@ class WelcomeMessage(Message):
 
     Attributes:
         content: The greeting text declared on the TeamCard.
-        display_type: Display category for UI rendering; defaults to "ai".
+        display_type: Display category for UI rendering; defaults to "other"
+            -- the greeting is a synthetic system announcement, not an AI turn.
     """
 
     content: str
-    display_type: Literal["human", "ai", "other"] = "ai"
+    display_type: Literal["human", "ai", "other"] = "other"

--- a/tests/integration/test_welcome_announcement.py
+++ b/tests/integration/test_welcome_announcement.py
@@ -144,7 +144,7 @@ class TestWelcomeAnnouncement:
         welcome = sent.message
         assert isinstance(welcome, WelcomeMessage)
         assert welcome.content == _GREETING
-        assert welcome.display_type == "ai"
+        assert welcome.display_type == "other"
 
     def test_no_announcement_when_welcome_message_none(
         self,

--- a/tests/models/test_welcome_message.py
+++ b/tests/models/test_welcome_message.py
@@ -18,10 +18,11 @@ class TestWelcomeMessage:
         """WelcomeMessage is a subclass of the core Message type."""
         assert issubclass(WelcomeMessage, Message)
 
-    def test_display_type_defaults_to_ai(self) -> None:
-        """display_type defaults to 'ai', overriding the base 'other' default."""
+    def test_display_type_defaults_to_other(self) -> None:
+        """display_type defaults to 'other' — the greeting is a synthetic system
+        announcement, not an AI turn (see ADR-17 amendment 2026-05-18)."""
         msg = WelcomeMessage(content="hi")
-        assert msg.display_type == "ai"
+        assert msg.display_type == "other"
 
     def test_content_is_required(self) -> None:
         """content is a required field — omitting it raises ValidationError."""
@@ -30,8 +31,8 @@ class TestWelcomeMessage:
 
     def test_display_type_can_be_overridden(self) -> None:
         """display_type accepts any value of the three-value Literal."""
-        msg = WelcomeMessage(content="hi", display_type="other")
-        assert msg.display_type == "other"
+        msg = WelcomeMessage(content="hi", display_type="ai")
+        assert msg.display_type == "ai"
 
     def test_round_trip_via_deserialize_object(self) -> None:
         """A WelcomeMessage round-trips through model_dump / deserialize_object.
@@ -45,7 +46,7 @@ class TestWelcomeMessage:
         assert type(restored) is WelcomeMessage
         assert restored == original
         assert restored.content == "Welcome to the team!"
-        assert restored.display_type == "ai"
+        assert restored.display_type == "other"
         assert restored.id == original.id
 
 

